src/components/Navigation/Navigation.js

```diff
@@ -67,7 +67,7 @@ const Navigation = () => {
             </List>
         </div>
     );
-
+ 
```

Comment:
remove spacing

Reply:
This has been done, will be included when test is finished and pull request is updated.